scripts/test/support.py

```diff
@@ -146,7 +146,10 @@ def to_end(j):
         elif chunk.startswith('(assert_invalid'):
             continue
         elif chunk.startswith(('(assert', '(invoke')):
-            ret[-1][1].append(chunk)
+            # ret may be empty if there are some asserts before the first
+            # module
+            if ret:
+                ret[-1][1].append(chunk)
```

Comment:
So we just ignore those asserts? Should we error out instead?

Reply:
No, they are valid. It's ok to have an assert without a module, something like `assert_invalid( some bad wast without a module reference )`

Reply:
Oh sorry, I misunderstood the question.

Yes, right now we just ignore them. I actually need this fixed for getting the modules. But yes, this means this method will ignore some asserts. I'll add a FIXME, as in theory we want that too.

Reply:
I made it return a None module in that case, which does seem more correct. Code calling this will need to handle that.